MobileCenter/MobileCenter/MSMobileCenter.m

```diff
@@ -202,7 +202,7 @@ - (BOOL)configure:(NSString *)appSecret {
 - (void)start:(NSString *)appSecret withServices:(NSArray<Class> *)services {
   @synchronized(self) {
     BOOL configured = [self configure:appSecret];
-    if (configured) {
+    if (configured && services) {
```

Comment:
This is not the right fix for the issue because the method is not supposed to be used for empty service list.
However, I like this fix better and just working as `configure` is harmless.